pallets/rc-migrator/src/lib.rs

```diff
@@ -121,6 +121,12 @@ pub enum MigrationStage<AccountId, BlockNumber, BagsListScore, AccountIndex, Vot
 	/// The migration has not yet started but will start in the next block.
 	#[default]
 	Pending,
+	/// The migration has been scheduled to start at the given block number.
+	Scheduled {
+		block_number: BlockNumber,
+	},
+	/// The migration is initializing.
+	Initializing,
```

Comment:
```suggestion
	WaitingForAhReadiness,
```

Reply:
may be I better update the doc? It will be more general and if needed we can include more into it.

Reply:
Yea, it is not really clear what is initalized and when that stage will be done.